controller/lib/core/sensors.cpp

```diff
@@ -137,8 +153,19 @@ SensorReadings Sensors::GetSensorReadings() {
   auto patient_pressure = ReadPressureSensor(PATIENT_PRESSURE);
   auto inflow_delta = ReadPressureSensor(INFLOW_PRESSURE_DIFF);
   auto outflow_delta = ReadPressureSensor(OUTFLOW_PRESSURE_DIFF);
-  VolumetricFlow flow =
-      PressureDeltaToFlow(inflow_delta) - PressureDeltaToFlow(outflow_delta);
+
+  dp_inhale = inflow_delta.cmH2O();
+  dp_exhale = outflow_delta.cmH2O();
+  pressure = patient_pressure.cmH2O();
+
+  VolumetricFlow inflow = PressureDeltaToFlow(inflow_delta);
+  VolumetricFlow outflow = PressureDeltaToFlow(outflow_delta);
+  VolumetricFlow flow = inflow - outflow;
+
+  flow_inhale = inflow.liters_per_sec() * 1000.0f;
+  flow_exhale = outflow.liters_per_sec() * 1000.0f;
+  flow_delta = flow.liters_per_sec() * 1000.0f;
```

Comment:
Would you be willing to add a `TODO(jlebar): add ml_per_sec()`?

Reply:
Good idea, I'll take care of that.